app/models/program.rb

```diff
@@ -113,4 +118,23 @@ def create_difficulty_levels
                                                 color: '#EF6E69')
     true
   end
+
+  ##
+  # Check if languages string has the right format. Used as validation.
+  #
+  def check_languages_format
+    # All white spaces are removed to allow languages to be separated by ',' and ', ' but only the first option is saved
+    return unless self.languages.present?
+    self.languages = self.languages.delete(' ').downcase
+    # We check if every language is a valid ISO 639-1 language
```

Comment:
This line does not say what happens to the following line! That comment should be above the line it describes.